### PR TITLE
Describe requester passing reserved parameter value

### LINKED ISSUE #1147: Describe requester passing reserved parameter value
It is currently not described that if neither `default` nor `fixed` are specified for a reserved parameter, the user (requester) is expected to pass the value as a parameter. See the relevant code block in the api3dao/ois repo:

https://github.com/api3dao/ois/blob/20192c541d152e9a90373e6af509656f72280ca4/src/ois.ts#L72-L79

### PR DESCRIPTION
**Pull request by dcroote**
Closes #1147

### CODE REVIEW

File: docs/ois/v1.4/ois.md [hunk 1]
```diff
@@ -473,8 +479,8 @@ If `fixed` is used, `default` has no effect and should not be used.
 #### 5.4.3. `default`
 
 (Optional) The default value for the reserved parameter. Used when no value is
-provided. If `default` is used, `fixed` must not be used because it would
-override the default value.
+provided by a requester. If `default` is used, `fixed` must not be used because
```
Comment (Siegrift) [inline docs/ois/v1.4/ois.md]: fyi: validator validates this, so I'd rephrase it to: `Only one of the default or fixed can be used.`

Comment (Siegrift) [reply docs/ois/v1.4/ois.md]: or maybe: `Cannot be used together with "fixed"` and similar message for the `fixed`.

**Review by Siegrift — APPROVED** [review]
👍 LGTM

File: docs/ois/v1.4/ois.md (hunk 1 above)
Comment (dcroote) [reply docs/ois/v1.4/ois.md]: Good call 👍  I updated the language for previous versions as well since this validation was present when OIS became a standalone package (v1.0.0)
